src/main/java/com/artipie/asto/rx/RxCopy.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.asto.rx;
+
+import com.artipie.asto.Key;
+import io.reactivex.Completable;
+import java.util.List;
+
+/**
+ * A reactive version of {@link com.artipie.asto.Copy}.
+ *
+ * @since 0.19
+ * @checkstyle NonStaticMethodCheck (500 lines)
+ * @checkstyle MemberNameCheck (500 lines)
+ * @checkstyle ParameterNameCheck (500 lines)
```

Comment:
@Sammers21 The same for checkstyle suppression, here at least NonStaticMethodCheck is not necessary to avoid.

Reply:
No, they could not be avoided until this class will be implemented. If I would remove it. qulice would complain:
```
 Checkstyle: src/main/java/com/artipie/asto/rx/RxCopy.java[68]: This method must be static, because it does not refer to "this" (NonStaticMethodCheck)

```